src/wp-includes/class-walker-nav-menu.php

```diff
@@ -39,6 +39,21 @@ class Walker_Nav_Menu extends Walker {
 		'id'     => 'db_id',
 	);
 
+	/**
+	 * The URL to the privacy policy page.
+	 *
+	 * @since 6.2.0
```

Comment:
```suggestion
	 * @since 6.8.0
```

Should be `6.8.0` instead of `6.2.0`